Cairo Desktop/Cairo Desktop/Cairo.xaml

```diff
@@ -3,7 +3,6 @@
     xmlns:cairo="clr-namespace:CairoDesktop" xmlns:l10n="clr-namespace:CairoDesktop.Localization;assembly=CairoDesktop.Localization">
     <!-- Menu resources -->
     <LinearGradientBrush StartPoint="0,0" EndPoint="0,1" x:Key="MenuBarBackground">
-        <GradientStop Color="#CF000000" Offset="0.0" />
```

Comment:
Was this intentional?

Reply:
This was not. Must have been unintentional. Not even sure what happened. I’ll add back and update

Reply:
Added back